src/server.h

```diff
@@ -1323,6 +1323,15 @@ struct redisMemOverhead {
     } *db;
 };
 
+/* Replication error behavior determines the replica behavior
+ * when it receives an error over the replication stream. In
+ * either case the error is logged. */
+enum {
+    PROPAGATION_ERR_BEHAVIOR_IGNORE = 0,
+    PROPAGATION_ERR_BEHAVIOR_PANIC,
+    PROPAGATION_ERR_BEHAVIOR_PANIC_ON_REPLICAS
```

Comment:
mmm maybe `PROPAGATION_ERR_BEHAVIOR_PANIC_ON_REPLICAS_ONLY`?

Reply:
@guybe7 Do you also think we should change the config variable name as well to 'panic-on-replicas-only' ?

Reply:
yes (but i'm not married to this suggestion)